### ISSUE TITLE
jdknext failing to compile, Could not find or load main class com.ibm.j9ddr.tools.ClassGenerator

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#11402]
https://ci.eclipse.org/openj9/job/Pipeline-OpenJDK-Acceptance/396/
```
01:01:12  Generating DDR pointer class source files
01:01:13  superset directory name : /home/jenkins/workspace/Build_JDKnext_s390x_linux_OpenJDK/build/linux-s390x-server-release/vm/
01:01:13  superset file name : superset.dat
01:01:13  Writing generated classes to /home/jenkins/workspace/Build_JDKnext_s390x_linux_OpenJDK/build/linux-s390x-server-release/support/ddr/gensrc/com/ibm/j9ddr/vm29/pointer/generated
01:01:14  Unhandled structure type: _IO_FILEPointer->_lock _IO_lock_t*
01:01:15  Processing complete
01:01:15  Generating DDR structure stub source files
01:01:16  superset directory name : /home/jenkins/workspace/Build_JDKnext_s390x_linux_OpenJDK/build/linux-s390x-server-release/vm/
01:01:16  superset file name : superset.dat
01:01:16  Generating DDR pointer and structure class files
01:01:16  Writing generated classes to /home/jenkins/workspace/Build_JDKnext_s390x_linux_OpenJDK/build/linux-s390x-server-release/support/ddr/gensrc/com/ibm/j9ddr/vm29/structure
01:01:16  Error: Could not find or load main class com.ibm.j9ddr.tools.ClassGenerator
01:01:16  Caused by: java.lang.ClassNotFoundException: com.ibm.j9ddr.tools.ClassGenerator
01:01:16  gmake[3]: *** [/home/jenkins/workspace/Build_JDKnext_s390x_linux_OpenJDK/build/linux-s390x-server-release/support/ddr/classes.done] Error 1
01:01:16  /home/jenkins/workspace/Build_JDKnext_s390x_linux_OpenJDK/closed/DDR.gmk:122: recipe for target '/home/jenkins/workspace/Build_JDKnext_s390x_linux_OpenJDK/build/linux-s390x-server-release/support/ddr/classes.done' failed
```

https://github.com/ibmruntimes/openj9-openjdk-jdk/compare/ac04db2e...0e3d888

I'm guessing the merge broke something related to building OpenJ9.

**Comment by JasonFengJ9** [comment]
It is related removal of `PATH_SEP`.

**Comment by tajila** [comment]
> It is caused by removal of PATH_SEP.

@JasonFengJ9 Do you have a link to that PR?

**Comment by JasonFengJ9** [comment]
@tajila it is https://github.com/ibmruntimes/openj9-openjdk-jdk/commit/9bac781cde77b39e4d380d6d3ed71d8ef052805f

**Comment by JasonFengJ9** [comment]
Created PR https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/255

**Comment by JasonFengJ9** [comment]
Somehow `closes eclipse/openj9#11402` didn't work for https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/255.
